### PR TITLE
Release of AB-BN on github

### PR DESCRIPTION
**Pull request by AaronHerrick**
This is release of AB-BN-30.  This is an afterburner improvement using a 5015 part fan.

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This is AB-BN-30, moving to usermods for public release
#### How was it tested? 
how much time have you got ? dozens of kilos of ABS and PLA, by many users.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Please see the requested changes below.

`wirestorage.png` is in your PR, but you don't have it in your README - either add it to your README, or remove the image file.

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+(special note:  I don't reccomend the Mosquito.  Its block design is problematic, but even more problematic are their business practices)
+
+# Assembly:
+
+![This is the fan after modification](ModifiedFan.png)
+
+Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting holes) off the 5015 fan, and take off the top cover. the ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. just make it look like the image above. If you cut too much its probably fine, as long as you don't damage the turbine.\
+\
+I have included stls for a de-earing tool. Its basically a handle/saw guide to put over the mounting tabs
```
[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: Added, thank you!

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+* no supports
+* 4 vertical shells
+* 5 solid layers top and bottom
+
+##### What files need to be printed?
+
+
+Everyone will need:
+
+* AB-BN-30_Front
```
[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: I've gone through and fixed the naming in the readme and the stls to be consistent, thank you!

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+* AB-BN-28_Spacer
+
+  ##### Depending on your hotend you will need:
+
+  #####   E3DV6:
+* AB-BN-28_Hotend-E3Dv6-Front
+* AB-BN-28_Hotend-E3Dv6-Back
+
+  ##### Dragon (high flow and regular):
+* AB-BN-30_Hotend-Dragon-Front
```
[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: resolved

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+* AB-BN-28_Hotend-Mos-Front
+* AB-BN-28_Hotend-Mos-Back
+
+(special note:  I don't reccomend the Mosquito.  Its block design is problematic, but even more problematic are their business practices)
+
+# Assembly:
+
+![This is the fan after modification](ModifiedFan.png)
+
+Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting holes) off the 5015 fan, and take off the top cover. the ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. just make it look like the image above. If you cut too much its probably fine, as long as you don't damage the turbine.\
```
[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: thank you for your attention to detail!

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+\
+I have included stls for a de-earing tool. Its basically a handle/saw guide to put over the mounting tabs
+
+Step 2: Fan test fit.\
+The fan fits into AB-BN-28-Front. But we have learned that different fans have very different shapes. AB-BN is designed to allow for adjustment for your fan. Here is how I do it:\
+\
+Test fit the fan in AB-BN-28_Front. Looking at it from the front, move the fan around to center the turbine blades. You may need to trim your fan housing a little more. Once the fan is centered, turn this over and note where you need to add shims between AB-BN-28-front and the fan.\
```
[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: fixed in 2 places

[the remark this answers is not in the document]
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: explained to use vhb tape

**Comment by AaronHerrick** [comment]
KiloQubit, thank you for your careful read through.  I think I resolved everything you found, please let me know if you have any further suggestions or comments

File: printer_mods/Badnoob/AB-BN/Readme.md [hunk 6]
```diff
@@ -114,61 +112,63 @@ I use the standard Voron print settings, but with 30% infill. I have gone as low
...
 
 # Assembly:
 
 ![This is the fan after modification](ModifiedFan.png)
 
-Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting holes) off the 5015 fan, and take off the top cover. the ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. just make it look like the image above. If you cut too much its probably fine, as long as you don't damage the turbine.\
+Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting tabs) off the 5015 fan, and take off the top cover. The ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. Just make it look like the image above. If you cut too much it's probably fine, as long as you don't damage the turbine.\
 \
-I have included stls for a de-earing tool. Its basically a handle/saw guide to put over the mounting tabs
+I have included stls for an optional de-earing tool (5015-deearing-tool and 5015-deearing-tool-b). The tool is meant as an handle and guide to saw off the ears. The tool fits around the ear and it's edge can be used as a saw guide to cut the mounting ear off.
```
Comment (KiloQubit) [inline printer_mods/Badnoob/AB-BN/Readme.md:141]: Replace `as an handle` with `as a handle`.

Replace `it's edge` with `its edge`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates!  One more change introduced in your update.

File: printer_mods/Badnoob/AB-BN/Readme.md (hunk 6 above)
Comment (AaronHerrick) [reply printer_mods/Badnoob/AB-BN/Readme.md]: corrected, thank you.

**Comment by KiloQubit** [comment]
Update `AB-BN-28_Hotend-E3Dv6-Back.stl` is now oriented incorrectly.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Comment by KiloQubit** [comment]
Conflicts fixed, squashed and merged in PR 304.

**Comment by KiloQubit** [comment]
@AaronHerrick Please delete your fork of VoronUsers, it is very old and causes merge conflicts.  I've fixed the conflict for this PR and merged the changes, but the same issue will happen with your next PR.  Easiest way to fix it is to delete your fork of VoronUsers, and make a new fork next time you want to make a PR.  Alternatively you could update your fork to get it up to date with upstream if you know git well enough.

**Pull request closed without merging**